monocle/gerrit/review.py

```diff
@@ -40,8 +40,8 @@ class ReviewesFetcher(object):
 
     log = logging.getLogger("monocle.ReviewesFetcher")
 
-    def __init__(self, host, repository_prefix):
-        self.host = host
+    def __init__(self, base_url, repository_prefix):
+        self.base_url = base_url
```

Comment:
This could be renamed to base_url in the CLI as well.

Reply:
done